package/src/mock-builders/event/connectionChanged.js

```diff
@@ -1,5 +1,6 @@
-export default (client) => {
+export default (client, online = true) => {
   client.dispatchEvent({
+    online,
```

Comment:
Did we add this here because we only knew that the connection changed but not the actual change state online/offline?